examples/python-tornado/server.py

```diff
@@ -81,5 +90,7 @@ def make_app():
     app.listen(8082)
     logging.critical("Listening on http://localhost:8080")
     loop = tornado.ioloop.IOLoop.current()
-    loop.call_later(0, init_table)
+    t = threading.Thread(target=update_thread)
```

Comment:
is a thread actually necessary for this?

Reply:
No, these updates are incomplete and have been used for testing. I've created a dedicated PR to track this and removed these changes #2659